src/lib.rs

```diff
@@ -18,21 +18,23 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+use crate::data::astarte::Astarte;
+use crate::data::Publisher;
+use crate::ota::ota_handler::OTAHandler;
+use crate::telemetry::TelemetryPayload;
```

Comment:
Why did you move here this import? IMHO the imports should be split into at least three groups, consisting of:
- imports from `std`
- imports from external crates
- imports from within the same crate

Refer to [StdExternalCrate](https://rust-lang.github.io/rustfmt/?version=v1.4.30&search=group#StdExternalCrate%5C%3A)
what do you think @bettio @rbino ?